src/components/Chat/AnonymousInputMessage.tsx

```diff
@@ -44,12 +42,6 @@ const styles = StyleSheet.create({
     justifyContent: 'center',
     marginLeft: 8
   },
-  enableButton: {
-    backgroundColor: colors.bondi_blue
-  },
-  disableButton: {
-    backgroundColor: colors.gray1
-  },
   previewPhotoContainer: {
     marginTop: 5,
     marginBottom: 5
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of `enableButton` and `disableButton` styles might affect the appearance of the button in different states. Ensure that these styles are not required elsewhere in your code or reintroduce them if necessary.
```diff
-  enableButton: {
-    backgroundColor: colors.bondi_blue
-  },
-  disableButton: {
-    backgroundColor: colors.gray1
-  },
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->